cequence/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dd3d5e86-3a2d-4859-a79b-3adb187955b1",
+  "app_id": "cequence",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your Cequence UAP events from your Preconfigured Datadog Dashboard",
+    "title": "Cequence",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Landing Page",
+        "image_url": "images/landing_page.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Tile Page",
+        "image_url": "images/total_transactions_and_mitigations.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Kubernetes",
+      "Category::Event Tracking",
+      "Category::Alerting",
+      "Category::Indicators of Compromise",
+      "Category::Notification",
+      "Category::Collaboration",
+      "Offering::Preconfigured Dashboard"
+    ]
```

Comment:
This integration only touches the Datadog dashboard. There no agent so this is information is not pertinent to this integration. It simply installs a "preconfigured dashboard" in Datadog so that customers can view their existing data that they are already sending to Datadog.